common/types/transaction_header.go

```diff
@@ -48,6 +47,5 @@ type LayerLimits struct {
 // Nonce is for ordering transactions.
 // TODO(dshulyak) we are using only counter until bitfield is defined.
 type Nonce struct {
-	Counter  uint64
-	Bitfield uint8
+	Counter uint64
```

Comment:
if we are doing without bitfield, can we make it a simple uint64 nonce instead of struct? 
readers that don't know the history will be confused.